running.py

```diff
@@ -0,0 +1,15 @@
+# Reference: https://replit.com/talk/learn/my-bot-goes-down-every-minute-and-doesn/11008/150912
```

Comment:
Please add a comment at the top of this file explaing why it was added.

Reply:
The explanation has been added